ls1intum/Artemis
# `Development`: Move translation keys for error messages into error.json

### Checklist
#### General
#### Client

### Motivation and Context
I noticed that there are two different locations in the current codebase where translations for error messages were stored. One was `error.json`, the other `global.json`. 

### Description
We should only have one general location for error messages. Therefore I moved all strings from `global.json` to `error.json`.

This PR additionally fixes a small bug with programming exercises and auxiliary repositories. The description was not sent to the server and therefore did not get saved.

### Steps for Testing
Try to invoke some server errors and check that the resulting alert shows correct strings in both German and English.

Additionally, test the saving of descriptions for auxiliary repositories:

Create and / or edit a programming exercise:

1. Add a new auxiliary repository to the exercise and set values for all fields.
2. Save the exercise and check that all values got saved.
3. Change the description to a long text (> 500 chars)
4. Make sure that a translated error message is shown. This message should also contain the maximum length of 500 chars.

### Review Progress





#### Code Review
#### Manual Tests
